cita-chain/src/main.rs

```diff
@@ -154,15 +154,15 @@ fn main() {
 
     let block_processor = BlockProcessor::new(Arc::clone(&chain), ctx_pub);
 
-    //chain 读写分离
-    //chain 读数据 => 查询数据
+    // Separation reading and writing process
+    // Handle Message
```

Comment:
first, it's not a translation, different meaning
sec, dispatch = handle?

Reply:
Done